build_tools/release_automation.py

```diff
@@ -0,0 +1,262 @@
+        {'name': 'Taylor, Jonathan', 'affiliation': 'Data Management and Software Centre, European Spallation Source ERIC', 'type':'ContactPerson'},
+        {'name': 'Udby, Linda', 'affiliation': 'Niels Bohr Institute', 'type':'ContactPerson'},
+        {'name': 'Weigandt, Katie', 'affiliation':'University of Washington','type':'Researcher'},
+    ],
+
+    'creators': [
+        {'affiliation': 'Oak Ridge National Laboratory', 'name': 'Doucet, Mathieu', 'orcid': '0000-0002-5560-6478'},
+        {'name': 'Cho, Jae Hie','affiliation': 'University of Tennessee Knoxville'},
+        {'name': 'Alina, Gervaise','affiliation': 'University of Tennessee Knoxville'},
+        {'name': 'Attala, Ziggy', 'affiliation': 'STFC - Rutherford Appleton Laboratory'},
+        {'name': 'Bakker, Jurrian','affiliation': 'Technical Unviersity Delft'},
+        {'name': 'Bouwman, Wim','affiliation': 'Technical Univeristy Deflt' },
+        {'name': 'Butler, Paul','affiliation': 'National Institute of Standards and Technology'},
+        {'name': 'Campbell, Kieran','affiliation': 'University of Oxford'},
+        {'name': 'Cooper-Benun, Torin', 'affiliation': 'STFC - Rutherford Appleton Laboratory'},
+        {'name': 'Durniak, Celine','affiliation': 'Data Management and Software Centre, European Spallation Source' },
+        {'name': 'Forster, Laura','affiliation': 'Diamond Light Source'},
+        {'name': 'Gonzales, Miguel','affiliation': 'Institut Laue-Langevin'},
+        {'name': 'Heenan, Richard','affiliation': 'STFC - Rutherford Appleton Laboratory',},
+        {'name': 'Jackson, Andrew','affiliation': 'European Spallation Source', 'orcid': '0000-0002-6296-0336'},
+        {'name': 'King, Stephen','affiliation': 'STFC - Rutherford Appleton Laboratory', 'orcid': '0000-0003-3386-9151'},
+        {'name': 'Kienzle, Paul','affiliation': 'National Institute of Standards and Technology'},
+        {'name': 'Krzywon, Jeff','affiliation': 'National Institute of Standards and Technology'},
+        {'name': 'Nielsen, Torben','affiliation': 'Data Management and Software Centre, European Spallation Source'},
+        {'name': "O'Driscoll, Lewis",'affiliation': 'STFC - Rutherford Appleton Laboratory'},
+        {'name': 'Potrzebowski, Wojciech','affiliation': 'Data Management and Software Centre, European Spallation Source ERIC', 'orcid': '0000-0002-7789-6779'},
+        {'name': "Prescott, Stewart",'affiliation': 'University of New South Wales'},
+        {'name': 'Ferraz Leal, Ricardo','affiliation': 'Oak Ridge National Laboratory'},
+        {'name': 'Rozycko, Piotr','affiliation': 'Data Management and Software Centre, European Spallation Source ERIC' },
+        {'name': 'Snow, Tim','affiliation': 'Diamond Light Source','orcid': '0000-0001-7146-6885'},
+        {'name': 'Washington, Adam','affiliation': 'STFC - Rutherford Appleton Laboratory'}
+        ],
+    'grants': [{'id': '10.13039/501100000780::654000'}],
```

Comment:
Should we not add SINE-2020 here?

Reply:
This should refer to SINE2020 grant but will double check